libraries/AP_Notify/Display.h

```diff
@@ -36,5 +37,8 @@ class Display: public NotifyDevice {
     uint8_t _mstartpos;
     uint8_t _movedelay;
     uint8_t _screenpage;
+
+    // stop showing text in display after this many millis:
+    const uint16_t _send_text_valid_millis = 10000;
```

Comment:
Why waste space if it fits in a smaller type? It will be cast automatically when needed.